oc_cdtapi/ForemanAPI.py

```
@@ -1183,15 +1185,23 @@ def get_host_uuid_v2(self, hostname):
         logging.debug('Reached get_host_uuid_v2')
         logging.debug('Passing to get_host_uuid_v1')
         return self.get_host_uuid_v1(hostname)
+    
+    def get_host_compute_attributes(self, hostname):
+        """
+        :param hostname: str
+        :return: dict
+        """
+        logging.debug('Reached get_host_compute_attributes')
```

Ruff (0.15.5)

[warning] `debug()` call on root logger

(LOG015)